docs/user-guide/authenticating-with-client-certificates.md

```diff
@@ -0,0 +1,82 @@
+# Authenticating with client certificates
+
+You can perform authentication with client certificates.
+
+If the keyring or a truststore contains at least one valid certificate authority (CA) other than the CA of API ML, it is possible to use the client certificates issued by this CA to authenticate to API ML. This feature is not enabled by default and must be configured.
```

Comment:
I don't fully understand this statement, wouldn't also a client cert signed by the CA of API ML be valid?

Reply:
I think it will be valid, this state is incorrect. Client certificate must be different from API ML but CA which singed it can be the same.